cove/templates/500.html

```diff
@@ -1,4 +1,4 @@
-{% extends 'base.html' %}
+{% extends app_base_template %}
```

Comment:
Should this be `request.current_app_base_template` for consistency with error.html etc.?